nucleus/admin/rest/rest-service/src/test/java/fish/payara/admin/rest/resources/CollectionLeafResourceTest.java

```diff
@@ -122,4 +122,29 @@ public void when_property_is_client_null_value_expect_add_to_map(){
         data8.put("-client", null);
         assertEquals(wrapper.getProcessedData(data8).get("id"), "-client");
     } 
+    
+    @Test
+    public void when_property_is_Xproperty_with_value_expect_add_to_map(){
+        Wrapper wrapper = new Wrapper();
+        Map<String, String> data9 = new HashMap<>();
+        data9.put("-XX:NewRatio=", "2");
+        assertEquals(wrapper.getProcessedData(data9).get("id"), "-XX:NewRatio=2");
```

Comment:
I think expected and actual are swapped here (and for the other tests).

Reply:
TestNG assert equals just likes to be different - caught me out the first time. 😄